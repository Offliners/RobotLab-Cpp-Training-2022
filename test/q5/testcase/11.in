(5+13/4*16+13*5+5/3*8*(8/8)-16-(14-2+19+7-2+11/8+10))*19*5*17*4*13=
