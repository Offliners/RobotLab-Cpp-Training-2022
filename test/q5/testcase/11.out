1
-4
-6
-204
